translations/ui-plugin-bursar-export/en.json

```diff
@@ -15,6 +21,130 @@
   "bursarExports.scheduling.time": "Start time",
   "bursarExports.scheduling.weekdays": "Run on weekdays",
 
+  "bursarExports.criteria.accordion": "Criteria",
+  "bursarExports.criteria.select.allOf": "All of",
+  "bursarExports.criteria.select.anyOf": "Any of",
+  "bursarExports.criteria.select.noneOf": "None of",
+  "bursarExports.criteria.select.age": "Age",
+  "bursarExports.criteria.select.amount": "Amount",
+  "bursarExports.criteria.select.owner": "Fee/fine owner",
+  "bursarExports.criteria.select.type": "Fee/fine type",
+  "bursarExports.criteria.select.location": "Item location",
+  "bursarExports.criteria.select.servicePoint": "Item service point",
+  "bursarExports.criteria.select.patronGroup": "Patron group",
+  "bursarExports.criteria.select.none": "No criteria (alwyays run)",
+
+  "bursarExports.criteria.age.value": "Older than (days)",
+  "bursarExports.criteria.type.automatic": "Automatic",
+  "bursarExports.criteria.location.inst": "Institution",
+  "bursarExports.criteria.location.camp": "Campus",
+  "bursarExports.criteria.location.lib": "Library",
+  "bursarExports.criteria.location.loc": "Location",
+  "bursarExports.criteria.servicePoint.value": "Service point",
+
+  "bursarExports.aggregate.accordion": "Aggregate by patron",
+  "bursarExports.aggregate.enabler": "Group data by patron",
+  "bursarExports.aggregate.description": "If enabled, each output row will correspond to a single patron with all of their accounts, rather than just a single account.",
+  "bursarExports.aggregate.filter": "Filter type",
+  "bursarExports.aggregate.filter.header": "Only include patrons with:",
+  "bursarExports.aggregate.filter.none": "None (include all patrons)",
+  "bursarExports.aggregate.filter.numAccounts": "Number of accounts",
+  "bursarExports.aggregate.filter.numAccounts.amount": "Number of accounts",
+  "bursarExports.aggregate.filter.totalAmount": "Total amount",
+  "bursarExports.aggregate.filter.totalAmount.amount": "Amount",
+  "bursarExports.aggregate.filter.operator": "Comparison operator",
+  "bursarExports.aggregate.filter.operator.less": "Less than but not equal to",
+  "bursarExports.aggregate.filter.operator.lessEqual": "Less than or equal to",
+  "bursarExports.aggregate.filter.operator.greater": "Greater than but not equal to",
+  "bursarExports.aggregate.filter.operator.greaterEqual": "Greater than or equal to",
+  "bursarExports.aggregate.filter.description": "This will be applied after accounts are evaluated per the \"Criteria\" specified above.",
```

Comment:
Please use "smart" quotes:
<img width="987" alt="image" src="https://github.com/ualibweb/ui-plugin-bursar-export/assets/8005215/6e0cec3e-4899-48a6-b049-b5eaad486f25">
You can encode these in JSON as follows:
```diff
-"bursarExports.aggregate.filter.description": "This will be applied after accounts are evaluated per the \"Criteria\" specified above.",
+"bursarExports.aggregate.filter.description": "This will be applied after accounts are evaluated per the \u201cCriteria\u201d specified above.",
```

This is also important since some other locales and languages use different quotes (lots of other languages use « text »)